### DISCUSSION TITLE
两个小问题

### DISCUSSION THREAD

**Discussion by Ackerven** [vuepress-theme-hope/vuepress-theme-hope#2714] [Q&A] [answered]
QQ 绑定的手机号找不回了，登不上QQ，在这里说下我这两天遇到的问题，我对前端不熟悉，不确实是 Bug 还是因为其他的原因。
1. 这个警告前面说 `tags` 已经弃用，后面又说使用 `tags` 来替代，应该是使用 `tag` 来替代
![image](https://user-images.githubusercontent.com/70303900/215084381-7cefb5e3-c008-4ea1-8c51-0535b6607c3a.png)
2. 使用 `vuepress-plugin-search-pro` 这个插件的时候，配置这里会报错，但运行是没问题的
![image](https://user-images.githubusercontent.com/70303900/215084810-c550e8f6-8abc-4bf5-b870-d1980c0ec723.png)

**Comment by Mister-Hope** [comment, marked as answer]
1. Fixed in https://github.com/vuepress-theme-hope/vuepress-theme-hope/commit/c6bbf2b7ba132bc96e30755a6d285e6f33fd83ef
2. Expected, the plugin has no idea what `frontmatter.category` type should be unless you tell it.

```ts
    searchProPlugin({
      indexContent: true,
      hotReload: true,
      customFields: [
        {
          getter: ({
            frontmatter,
          }: Page<Record<string, unknown>, { tag?: string[] }>) =>
            frontmatter.tag || null,
          formatter: `Tag: $content`,
        },
      ],
    }),
```

```ts
    searchProPlugin({
      indexContent: true,
      hotReload: true,
      customFields: [
        {
          getter: ({ frontmatter }) => frontmatter.tag as string[],
          formatter: `Tag: $content`,
        },
      ],
    }),
```

**Reply by Ackerven** [reply]
I get it. 👍
